docs/src/main/sphinx/release/release-455.md

```diff
@@ -0,0 +1,61 @@
+# Release 455 (28 Aug 2024)
+
+## General
+
+* Add query starting time in `QueryStatistics` in all [](admin-event-listeners).
+  ({issue}`23113`)
+* {{breaking}} Add JMX metrics for the bean
+  `trino.execution.executor.timesharing:name=TimeSharingTaskExecutor` replacing
+  metrics previously found in `trino.execution.executor:name=TaskExecutor`.
+  ({issue}`22914`)
+* Fix memory tracking issue for aggregations that could cause worker crashes
+  with out-of-memory errors. ({issue}`23098`)
+* Fix loading of custom `CatalogStore` implementations. Set `catalog.store` to
+  the name of the custom `CatalogStoreFactory`. `catalog-store.name` in
+  `etc/catalog-store.properties` is no longer required. ({issue}`22554`)
+* For file-based catalog store the relevant configuration now needs to be done
+  in `etc/catalog-store.properties` instead of `config.properties`.
+  ({issue}`23115`)
+
+## Delta Lake connector
+
+* Allow configuring endpoint for the native Azure filesystem. ({issue}`23071`)
+* Improve Glue connections with use exponential backoff strategy. ({issue}`23039`)
+* Fix query failures due to "Multiple entries with same key" when reading
+  Parquet files. ({issue}`23050`)
+
+## ClickHouse connector
+
+* Add support for creating tables with the `MergeTree` engine without the
+  `order_by` table property. ({issue}`23048`)
+
+## Hive connector
+
+* Allow configuring endpoint for the native Azure filesystem. ({issue}`23071`)
+* Improve Glue connections with use exponential backoff strategy. ({issue}`23039`)
+* Fix query failures due to "Multiple entries with same key" when reading
+  Parquet files. ({issue}`23050`)
+
+## Hudi connector
+
+* Allow configuring endpoint for the native Azure filesystem. ({issue}`23071`)
+
+## Iceberg connector
+
+* Allow configuring endpoint for the native Azure filesystem. ({issue}`23071`)
+* Improve Glue connections with use exponential backoff strategy. ({issue}`23039`)
+* Fix `$files` table not showing delete files with the Iceberg v2 format. ({issue}`16233`)
+
+## OpenSearch connector
+
+* Improve performance of queries that reference nested fields from OpenSearch
+  documents. ({issue}`22646`)
+
+## Prometheus connector
+
+* Add support for the catalog session properties `query_chunk_size_duration` and
+  `max_query_range_duration`. ({issue}`22319`)
+
+## Redshift connector
+
+* Add support for propagating Trino query cancellation to Redshift. ({issue}`22774`)
```

Comment:
That's an implementation detail. What's the user visible impact?

Reply:
Well.. cost and performance savings on Redshift I assume.. and the PR also mentions some Redshift logging differences in the query history but I think that is too low level. 

Should we reword somehow ? or remove if you see it only as a pure implementation detail

Reply:
Maybe say something like:

> * Release resources in Redshift promptly when a query is cancelled.